packages/site/src/content/docs/rules/ts/nonOctalDecimalEscapes.mdx

```diff
@@ -63,6 +63,7 @@ This rule is not configurable.
 
 ## When Not To Use It
 
+TODO "always be enabled"
```

Comment:
I'm going to file a followup for cleaning these AI slop descriptions that crept in.

Reply:
#1017 -> #1018